src/components/Modal/index.stories.js

```diff
@@ -128,7 +128,7 @@ class ModalExample extends Component {
                             future purchases.
                           </p>
 
-                          <div className='row small-gutters no-gutters-vertical'>
+                          <div className='row small-gutters no-gutters'>
```

Comment:
no-gutters also remove horizontal gutters, which i don't believe is the goal of the existing code

Reply:
it's a valid point.  I'll check and see if it implodes if removed